packages/commercetools/helpers/src/_utils.ts

```diff
@@ -0,0 +1,41 @@
+
+const getAttributeValue = (attribute) => {
+  switch(attribute.__typename) {
+    case 'StringAttribute':
+      return attribute.stringValue
+    case 'DateAttribute':
+      return attribute.dateValue
+    case 'DateTimeAttribute':
+      return attribute.dateTimeValue
+    case 'TimeAttribute':
+      return attribute.timeValue
+    case 'NumberAttribute':
+      return attribute.numberValue
+    case 'EnumAttribute':
+      return attribute.label
+    case 'LocalizedEnumAttribute':
+      return attribute.localizedLabel
+    case 'LocalizedStringAttribute':
+      return attribute.localizedString
+    case 'MoneyAttribute':
+      return attribute.centAmount
+    case 'BooleanAttribute':
+      return attribute.booleanValue
+    case 'ReferenceAttribute': 
+      return { typeId: attribute.typeId, id: attribute.id }
+    default:
+      return null
+  }
+}
+
+const formatAttributeList = (attributes: any) => {
```

Comment:
what about this approach?
```TS
export const formatAttributeList = (attributes: object[]) => attributes.map(.....)
```

Reply:
yup, I forgot typings in few places ;) actually there is a typo for that